odd-platform-api/src/main/java/org/opendatadiscovery/oddplatform/mapper/DataEntityMapperImpl.java

```diff
@@ -402,9 +403,14 @@ public DataEntityRef mapRef(final DataEntityPojo pojo) {
     @Override
     public DataEntityUsageInfo mapUsageInfo(final DataEntityStatisticsPojo pojo,
                                             final Long filledEntitiesCount) {
-        final Map<Integer, Long> classesCount = JSONSerDeUtils.deserializeJson(pojo.getDataEntityClassesCount().data(),
-            new TypeReference<>() {
-            });
+        final Map<Integer, Long> classesCount;
+        if (pojo.getDataEntityClassesCount() == null) {
+            classesCount = new HashMap<>();
```

Comment:
this way you'd need to perform several pointless lookups when you know for sure there'd be a zero.